### ISSUE TITLE
It looks like `coveralls-api-result` is no longer used

### ISSUE THREAD

**Issue by StummeJ** [coverallsapp/github-action#193]
As the title says, the v2 tags look like the `coveralls-api-result` output is no longer used, giving a false pretense that we can act based on the results of the action

**Comment by muneeb706** [comment]
There is a troubleshooting section in the README that addresses this issue. After going through the steps mentioned, I was able to view the coveralls api response in logs. In my case, I had to enable LEAVE COMMENTS in repo settings page in Coveralls app.

However, after migrating to V2 from V1. I am unable to coverage results in PR conversation tab.

**Comment by fletort** [comment]
Yes the output of the action seems not available

**Comment by StummeJ** [comment]
Closing since it's no longer specified in the action

**Comment by fletort** [comment]
But it is specified in the readme... so they are a problem....  
Remove from the Readme, or make working again this output that can be very interesting....

**Comment by afinetooth** [comment]
@fletort @StummeJ @muneeb706 I have commented out the outdated `v1.x` content re: `coveralls-api-result` from the current [README](https://github.com/coverallsapp/github-action/blob/main/README.md) (for `v2`).

I am not personally familiar with `coveralls-api-result` from `v1` as our sole output option, but if you feel it should be part of the action, please open a new issue with the request, and anything you can share about how it _used_ to work, or how it _should_ work in your opinion, and about your use case, would be a big help. 🙏

**Comment by muneeb706** [comment]
@afinetooth

I experienced an issue while trying to view coverage details in the GitHub Pull Request's conversation tab. After investigation, I discovered that the problem was in my configuration - I had been pointing to the master branch instead of specifying a particular version. Since I was expecting v1 behavior while the repository had been updated to v2, some recent updates were not incorporated into my scripts. After explicitly switching to v2 in my configuration, the coverage details are now displaying correctly in the PR conversation tab, and everything is working as expected. 

Thank you
